README.md

```diff
@@ -228,253 +376,406 @@ Your directory structure should look like this:
+"http://localhost:8080/football-team". Then click "Send" to post.
 
-```java
-spring.jpa.hibernate.ddl-auto=none
-```
+![Post-Football-Team](https://curriculum-content.s3.amazonaws.com/spring-mod-1/dto/postman-add-football-team.png)
 
-## Read
+Now let's try sending a GET request using the path
+"http://localhost:8080/football-team/Dallas-Cowboys":
 
-We will be using the `findById` and the `findAll` method. Update the `run`
-method in the `SpringDataJpaExampleApplication` class:
+![Get-Football-Team](https://curriculum-content.s3.amazonaws.com/spring-mod-1/dto/postman-get-footbal-team.png)
 
-```java
-public void run(String... args) {
-	Optional<Teacher> teacher1 = repository.findById(1);
-	System.out.println("Teacher with ID 1:" + teacher1.orElse(null));
+The big test will be what happens when we stop the application. Let's try
+restarting it! But before we completely start up the application again, navigate
+back to the `application.properties` file and set the property
+`spring.jpa.hibernate.ddl-auto` to `none`:
 
-	Iterable<Teacher> teachers = repository.findAll();
-	System.out.println("All Teachers:");
-	teachers.forEach(System.out::println);
-}
+```properties
+spring.jpa.hibernate.ddl-auto=none
 ```
 
-You should see the following output in the console for the `findById` call:
+Now start the application back up again and try sending the GET request again.
+We should still get the same result as we did last time! If we open up pgAdmin4
+and navigate to our sports database, we can even see the data there still!
 
-```
-Hibernate: select teacher0_.id as id1_0_0_, teacher0_.name as name2_0_0_ from teacher teacher0_ where teacher0_.id=?
-Teacher with ID 1:Teacher{id=1, name='Otha'}
-```
+![data-persisted](https://curriculum-content.s3.amazonaws.com/spring-mod-1/repository/pgadmin-select-all-1.png)
 
-For the `findAll` call the output should look similar to this:
+## Updating and Deleting
 
-```
-Hibernate: select teacher0_.id as id1_0_, teacher0_.name as name2_0_ from teacher teacher0_
-All Teachers:
-Teacher{id=1, name='Otha'}
-Teacher{id=2, name='Ian'}
-```
+So far, our example only performs a POST and a GET request. But what if we wanted
+to perform an UPDATE or DELETE request as well?
 
-We can see that Spring can automatically create the proper Hibernate queries and
-fetch the desired resources from the database.
+Let's modify the code to do so!
 
-## Update
+In the `FootballService` class, add the following methods:
 
-In order to update database records, we have to first fetch them using
-`findById`, update the object instance, and call the the `repository.save`
-method with the modified object instance.
+```java
+    public String updateFootballTeam(Integer id, FootballTeamDTO footballTeamDTO) {
+        Optional<FootballTeam> optionalFootballTeam = footballRepository.findById(id);
+        if (optionalFootballTeam.isPresent()) {
+            FootballTeam footballTeamEntity = optionalFootballTeam.get();
```

Comment:
I think you could use ModelMapper here, too.
```modelMapper.map(footballTeamDTO, footballTeamEntity);```
I *believe* that will take the properties from the dto and put them into the entity.